subprojects/language-java/src/main/java/org/gradle/api/tasks/javadoc/Javadoc.java

```diff
@@ -102,6 +102,7 @@
  * }
  * </pre>
  */
+@SuppressWarnings("MethodMayBeStatic")
```

Comment:
:x: Don't suppress warnings at class level for a single method affected. Also, why not make it static?